tests/integration_tests/test_hyperopt.py

```diff
@@ -123,30 +118,9 @@ def _setup_ludwig_config(dataset_fp: str) -> Tuple[Dict, str]:
     return config, rel_path
 
 
-@contextlib.contextmanager
-def ray_start(num_cpus: Optional[int] = None, num_gpus: Optional[int] = None):
-    res = ray.init(
-        num_cpus=num_cpus,
-        num_gpus=num_gpus,
-        include_dashboard=False,
-        object_store_memory=150 * 1024 * 1024,
-    )
-    try:
-        yield res
-    finally:
-        ray.shutdown()
-
-
-@pytest.fixture(scope="module")
-def ray_cluster():
-    gpus = [i for i in range(torch.cuda.device_count())]
-    with ray_start(num_gpus=len(gpus)):
-        yield
-
-
```

Comment:
Switched to using the fixtures in conftest.py to ensure that the ray cluster was started with `hyperopt_resources: 1000`.